lib/components/formik/Radio.js

```diff
@@ -6,13 +6,14 @@ const RadioGroup = ({
   label,
   name,
   options,
-  className,
-  childrenClassName,
-  stacked,
+  className = "",
+  childrenClassName = "",
+  stacked = false,
+  customOption = null,
```

Comment:
Do we need this customOption?
@vinay0x @edwinbbu Please share your thoughts

Reply:
It's a good feature in my opinion.

But the naming is not. It doesn't convey it's usecase in one look. 

Can we name this `defaultChecked` or `defaultOption`, if we are planning to keep this attr? Let me know your thoughts.

Reply:
@goutham-subramanyam I don't think it's a needed feature. I can't think of a situation where it is necessary. We can remove it.